packages/fsml/fsml.0.2a/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis:
+  "A library for describing and describing synchronous finite state machines"
+description:
+  "A library for describing and describing synchronous finite state machines"
+maintainer: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+authors: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+license: "MIT"
+homepage: "http://github.com/jserot/fsml"
+doc: "http://jserot.github.io/fsml"
+bug-reports: "jocelyn.serot@uca.fr"
+depends: [
+  "dune" {>= "2.6" & >= "2.0"}
```

Comment:
Could you remove the `(dune (>= 2.0))` from the `dune-project` file? When requiring dune 2.6 it will already be added.

Reply:
Done